docs/sphinx/install.rst

```diff
@@ -1,128 +1,183 @@
-CUDA Quantum Open Beta Installation
+Getting Started
 *******************************************
 
-Docker Image
---------------------
+This guide walks through how to :ref:`install CUDA Quantum <install-cuda-quantum>` on your system, and how to set up :ref:`VS Code for local development <local-development-with-vscode>`.
+The section on :ref:`connecting to a remote host <connect-to-remote>` contains some
+guidance for application development on a remote host where CUDA Quantum is installed.
 
-Install the Docker Image
-++++++++++++++++++++++++++++++++++++
+.. _install-cuda-quantum:
 
-Docker images for all CUDA Quantum releases are available on the `NGC Container Registry`_.
-To download images from NGC, please follow the following steps if you have not done so already:
+Local Installation
+------------------------------------
 
-- `Create an account <https://ngc.nvidia.com/signin>`__
-- `Sign in <https://ngc.nvidia.com/signin>`__ to access your account and go to `Setup <https://ngc.nvidia.com/setup>`__.
-- Click on `Get API Key` and generate a new key (this will invalidate any existing keys).
-- Follow the instructions that appear to use that key to log in to the NGC registry using Docker.
+A fully featured CUDA Quantum installation including all C++ and Python tools is available as a 
+`Docker image`_. A `Singularity`_ container can easily be created based on these images. Additionally, we distribute pre-built `Python wheels`_ via PyPI.
+If you would like to build CUDA Quantum from source instead, please follow the instructions on the `CUDA Quantum GitHub repository`_.
 
-Once you have done so, run `docker login nvcr.io` (again) to confirm you can authenticate with the registry. 
-You should see a message "Login Succeeded".
+.. _CUDA Quantum GitHub repository: https://github.com/NVIDIA/cuda-quantum/blob/main/Building.md
 
-.. _NGC Container Registry: https://catalog.ngc.nvidia.com/orgs/nvidia/containers/cuda-quantum
+If you are unsure which option suits you best, we recommend using our `Docker image`_ to develop your applications in a controlled environment that does not depend on, or interfere with, other software
+that is installed on your system.
+
+.. _install-docker-image:
 
-In addition to publishing stable releases, we also publish docker images whenever we update 
-certain branches of our `GitHub repository <https://github.com/NVIDIA/cuda-quantum>`_.
-These images are published in a separate location `nvidia/nightly` on NGC, as well as on GitHub.
+Docker image
```

Comment:
I suggest to rename to 'Docker' only (remove the word image) as this section also talks about Docker containers and Docker engine

Reply:
done